models/comments-models.js

```diff
@@ -3,12 +3,27 @@ const db = require("../db/connection");
 function fetchCommentsByArticleId(article_id) {
   return db
     .query(
-      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`,
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`,
       [article_id]
     )
     .then(({ rows: comments }) => {
       return comments;
     });
 }
 
-module.exports = { fetchCommentsByArticleId };
+function addNewComment(article_id, username, body) {
+  return db
+    .query(
+      `INSERT INTO comments
+          (article_id, author, body )
+          VALUES
+          ($1,$2,$3)
+                 RETURNING *;`,
+      [article_id, username, body]
+    )
+    .then(({ rows }) => {
+      return rows;
```

Comment:
If we can only post 1 comment at a time, maybe it's not necessary to send back an array (arrays are usually for multiple elements)